modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -362,7 +362,7 @@ export default {
     },
     // Retrieve table heading fields from the schema, based on the currently
     // opened item. Available only when the field style is `table`.
-    visibleSchema() {
+    visibleSubFields() {
```

Comment:
I think this was correct as `visibleSchema`, which we already use as the plural of fields everywhere.